snipper/snippet.py

```diff
@@ -94,7 +94,7 @@ def get_detail_for_print(self):
             u'Title     : {}'.format(self.data['title']),
             u'Is private: {}'.format("Yes" if self.data['is_private'] else "No"),
             u'Url       : {}'.format(self.data['links']['html']['href']),
-            u'Owner     : {}'.format(self.data['owner']['username']),
```

Comment:
I think displaying `nickname` is good. What you think about displaying it as `display_name (nickname)`?